### ISSUE TITLE
6.5 RC1 Chooser unable to delete unwanted remote clients

### ISSUE THREAD

**Issue by NameOfTheDragon** [ASCOMInitiative/ASCOMPlatform#28]
In the following screen snipping, notice "ASCOM Remote Client 1" in the drop-down list.

![image](https://user-images.githubusercontent.com/1268490/75421208-a2ee0980-5931-11ea-948d-8c2aecb8af68.png)

I wanted to delete this remote client, but the delete screen is empty:
![image](https://user-images.githubusercontent.com/1268490/75421381-08da9100-5932-11ea-973a-9b4cf8b5111c.png)

**Comment by Peter-Simpson** [comment]
I did think about this at one point but triaged it out when I ran out of time. 

I've now added the capability to list ASCOM Remote drivers as well as Alpaca Dynamic drivers to the client manager. This is the only change from RC1, does it work for you?

Installers:
[ASCOM Platform](https://download.ascom-standards.org/beta/ASCOMPlatform65.exe)
[Developer Tools](https://download.ascom-standards.org/beta/ASCOMPlatform65Developer.exe)
[Remove ASCOM](https://download.ascom-standards.org/beta/RemoveASCOM.zip)
